homeassistant/components/flexmeasures/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "flexmeasures",
+  "name": "FlexMeasures",
+  "codeowners": ["@flix6x"],
+  "config_flow": true,
+  "dependencies": ["http"],
+  "documentation": "https://www.home-assistant.io/integrations/flexmeasures",
+  "homekit": {},
+  "iot_class": "cloud_polling",
+  "requirements": ["flexmeasures-client==0.1.8", "isodate==0.6.1"],
+  "ssdp": [],
+  "zeroconf": []
```

Comment:
Remove zeroconf and ssdp

Reply:
removed: 30ec2044247e56aff10994d90b8929b423284e29